viewer/core/src/glsl/post-processing/outline-detect-combine.frag

```diff
@@ -71,6 +71,11 @@ void main() {
   float backDepth = texture2D(tBackDepth, vUv).r;  
   float customDepth = texture2D(tCustomDepth, vUv).r;
   float ghostDepth = texture2D(tGhostDepth, vUv).r;
+
+  customDepth = customDepth > 0.0 ? customDepth : 1.0; 
+  backDepth = backDepth > 0.0 ? backDepth : 1.0; 
+  ghostDepth = ghostDepth > 0.0 ? ghostDepth : 1.0;
+  frontDepth = frontDepth > 0.0 ? frontDepth : 1.0; 

```

Comment:
add a comment explaining this (yes - it should have been there in the code you removed too, but hey - never to late to improve)